cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/Measurement.java

```diff
@@ -91,20 +92,39 @@ private Measurement(final String name, final double value, final long time, fina
      * @return Measurement object
      */
     public static Measurement create(final String name, final double value, final long time, final TimeUnit timeUnit,
-            final Map<String, String> tags, final Map<String, String> fields) {
+                                     final Map<String, String> tags, final Map<String, String> fields) {
         return new Measurement(name, value, time, timeUnit, tags, fields);
     }
 
     @Override
     public String toString() {
```

Comment:
toString() is too broad and you should not rely on it to return json. Make it a separate method.